tests/TestDelete.m

```diff
@@ -46,20 +46,21 @@ function TestDelete_testTwoFKOnePK(testCase)
             dj.createSchema(package,[testCase.test_root '/test_schemas'], ...
                 [testCase.PREFIX '_testlab']);
 
-            users = [{'user0'; 'user1'; 'user2'}];
+            users = [{'user0'; 'user1'; 'user2'; 'user3'}];
 
             insert(TestLab.User, users);
 
-            duty = [{'2020-01-01','user0','user1'},
-                    {'2020-01-02','user1','user2'},
-                    {'2020-12-31','user0','user2'}];
+            duty = [{'2020-01-01','user0','user2', 'user1'},
+                    {'2020-01-02','user0','user1', 'user2'},
+                    {'2020-12-30','user1','user2', 'user0'},
+                    {'2020-12-31','user0','user2', 'user3'}];
 
             insert(TestLab.Duty, duty);
 
             key.user_id = 'user1';
             del(TestLab.User & key);
 
-            testCase.verifyEqual(length(fetch(TestLab.User)), 2);
+            testCase.verifyEqual(length(fetch(TestLab.User)), 3);
             testCase.verifyEqual(length(fetch(TestLab.Duty)), 1);
             testCase.verifyEqual(length(fetch(TestLab.User & 'user_id = "user1"')), 0);
             testCase.verifyEqual(length(fetch(TestLab.Duty & 'duty_first = "user1" OR duty_second = "user1"')), 0);
```

Comment:
```suggestion
            testCase.verifyEqual(length(fetch(TestLab.Duty & 'duty_first = "user1" OR duty_second = "user1" OR duty_third = "user1"')), 0);
```

We can also now modify the name of this function from `TestDelete_testTwoFKOnePK` -> `TestDelete_testThreeFKOnePK`.